modules/fluid_properties/doc/content/documentation/systems/Modules/FluidProperties/TabulatedFluidProperties.md

```diff
@@ -112,6 +112,14 @@ All fluid properties read from a file or specified in the input file (and their
 respect to pressure and temperature) will be calculated using bicubic interpolation, while all
 remaining fluid properties will be calculated using the provided FluidProperties UserObject.
 
+## Note for Porous Flow users
+
+In the [Porous Flow](/modules/porous_flow/index.md) module, fluid density and viscosity are typically
+calculated at the same time. To achieve the maximum benefit from `TabulatedFluidProperties`, users
+should ensure that both density and viscosity are calculated by interpolation, by either providing
+them in the supplied properties file, or making sure that both are specified in the
+`interpolated_properties` input parameter if no data file exists.
```

Comment:
May be we could solve this by either:

1. Generalize this note by saying something like "Performance note" and describe how to achieve the maximum benefits
2. Move this note to `porous_flow`. Probably to some place where you discuss the usage of `TabulatedFluidProperties`.

Reply:
That would solve it, but the bigger issue is whether we should allow any kind of cross-listing. It seems to me that a good system would allow you to talk about capabilities elsewhere. I also understand the problems that can arise from that strategy. Still, I'd like to brainstorm ways to keep these cross-listings when possible.

Reply:
Thanks guys. I'll just move this to the part of the `porous_flow` documentation where it discusses using `fluid_properties`.